lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_bump_version_hotfix.rb

```diff
@@ -9,14 +9,18 @@ def self.run(params)
         create_config(params[:previous_version], params[:version])
         show_config()
 
-        UI.message 'Updating Fastlane deliver file...'
-        Fastlane::Helper::Ios::VersionHelper.update_fastlane_deliver(@new_short_version)
-        UI.message 'Done!'
+        update_deliverfile = params[:skip_deliver] == false
```

Comment:
As per comment in the code below, I decided to call this parameter "skip" instead of "include" because the majority of the consumers still have `Deliverfile`.

When they'll all have moved to not use `Deliverfile`, we shall remove the parameter altogether.